src/components/videoPage/VideoAIMetadataContent.tsx

```diff
@@ -0,0 +1,50 @@
+import { LoadingOutlined } from '@ant-design/icons';
+import s from 'src/components/videoPage/style.module.less';
+import Badge from '@boclips-ui/badge';
+import { Typography } from '@boclips-ui/typography';
+import React from 'react';
+
+interface Props {
+  isLoading: boolean;
+  metadata?: string[];
+  type: string;
+}
+export const VideoAIMetadataContent = ({
+  isLoading,
+  metadata,
+  type,
+}: Props) => {
+  const getAIBadge = () =>
+    isLoading ? (
+      <LoadingOutlined className={s.spinner} />
+    ) : (
+      <Badge value="AI generated" />
+    );
+
+  const showContent = () => (
+    <ul className={s.outcomeList}>
+      {metadata.map((elem: string) => (
+        <Typography.Body as="li" size="small" className="text-gray-800">
+          {elem}
+        </Typography.Body>
+      ))}
+    </ul>
+  );
+
+  const showErrorMessage = () => (
+    <Typography.Body as="p" size="small" className="text-gray-800">
+      No {type} found.
+    </Typography.Body>
+  );
+
+  const content = metadata ? showContent() : showErrorMessage();
```

Comment:
consider using  `metadata && metadata.length > 0` as checking empty array with `?` will return true

also we could use component here like <ShowContent /> and <ShowErrorMessage />

Reply:
hmm, that could be expected behaviour... do we assume that if there is no data we show error message?